engine/battle/core.asm

```diff
@@ -2493,9 +2493,9 @@ MoveSelectionMenu:
 	   ; so it is necessary to put the di ei block to not cause tearing
 	call TextBoxBorder
 	hlcoord 4, 12
-	ld [hl], $7a
+	ld [hl], "─" ; bottom PrintMenuItem window border
 	hlcoord 10, 12
-	ld [hl], $7e
+	ld [hl], "┘" ; right corner of the border
```

Comment:
This is like `ld a, NUM_MOVES ; number of moves`.

Reply:
Eh, "┘" meaning "right corner of bottom window border with a Poké Ball on top" is not that obvious, but sure.

Is the comment immediately before fine or should I remove it as well?

Reply:
Yeah, remove that one too. Charmaps in general are taken to be self-explanatory, just like constant and label names; if we feel otherwise, the charmap should be changed, e.g. `"<TEXTBOX_BOTTOM_RIGHT>"`. (which, just no, lol)